packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -83,6 +82,8 @@ class Dismissible extends StatefulWidget {
     this.direction: DismissDirection.horizontal,
     this.resizeDuration: const Duration(milliseconds: 300),
     this.dismissThresholds: const <DismissDirection, double>{},
+    this.rollbackDuration: const Duration(milliseconds: 300),
```

Comment:
this should be 200ms, right? To match the previous _kDismissDuration?

Reply:
Yes, you are right.